sub-packages/bionemo-scdl/tests/bionemo/scdl/io/test_single_cell_memmap_dataset.py

```diff
@@ -122,40 +121,6 @@ def test_h5ad_no_file(tmp_path):
         ds.load_h5ad(anndata_path=tmp_path / "a")
 
 
-def test_swap_mmap_array_result_has_proper_length(tmp_path, create_and_fill_mmap_arrays):
```

Comment:
I suggest adding some of these tests for correct length of the underlying arrays back to the test set after calling the codepath utilizing `extend`.

Reply:
good idea!